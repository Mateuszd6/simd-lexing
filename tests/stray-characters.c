foobar

int
main()
{

}

long long long long long long long long long long long long long long long long
long long long long long long long long long long long long long long long long
