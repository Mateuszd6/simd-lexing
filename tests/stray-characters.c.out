../tests/stray-characters.c:1:1: TOK 0 (L = 6): "foobar"
../tests/stray-characters.c:3:1: TOK 0 (L = 3): "int"
../tests/stray-characters.c:4:1: TOK 0 (L = 4): "main"
../tests/stray-characters.c:4:5: TOK 1 (L = 1): "("
../tests/stray-characters.c:4:6: TOK 1 (L = 1): ")"
../tests/stray-characters.c:5:1: TOK 1 (L = 1): "{"
../tests/stray-characters.c:7:1: TOK 1 (L = 1): "}"
../tests/stray-characters.c:9:1: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:6: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:11: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:16: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:21: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:26: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:31: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:36: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:41: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:46: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:51: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:56: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:61: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:66: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:71: TOK 0 (L = 4): "long"
../tests/stray-characters.c:9:76: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:1: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:6: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:11: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:16: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:21: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:26: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:31: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:36: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:41: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:46: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:51: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:56: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:61: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:66: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:71: TOK 0 (L = 4): "long"
../tests/stray-characters.c:10:76: TOK 0 (L = 4): "long"
Parsed: 11 lines, 35 ids, 0 strings, 0 chars, 0 ints, 0 hex,   0 floats,    0 //s, 0 /**/s,   0 #foo
